frontend/src/_includes/partials/filter-section.html

```diff
@@ -0,0 +1,22 @@
+<div x-data="{ expanded: false }" class="mb-2">
+  <div
+    @click="expanded = ! expanded"
```

Comment:
We don't keep space after the `!` operator. Below also.

Reply:
updated